src/main/java/smartthings/konsumer/KafkaListener.java

```diff
@@ -34,6 +40,7 @@ public KafkaListener(ListenerConfig config) {
 		consumer = Consumer.createJavaConsumerConnector(config.getConsumerConfig());
 		topic = config.getTopic();
 		streamProcessor = new StreamProcessorFactory(config).getProcessor();
+		circuitBreaker = new SimpleCircuitBreaker();
```

Comment:
I can't really think of any good reason to customize the circuit breaker. Most of the functionality would end up in the filters anyway. What are you thinking the builder should look like? I'm happy to make the change, just want to make sure I understand what you are thinking here.